params.go

```diff
@@ -305,11 +313,13 @@ func (s *SUint64) String() string {
 // It's a comma-separated list, so we split it.
 func (s *SUint64) Set(value string) error {
 	for _, dt := range strings.Split(value, ",") {
-		parsed, err := strconv.ParseFloat(dt, 64)
-		if err != nil {
-			return err
+		if len(dt) > 0 {
+			parsed, err := strconv.ParseFloat(dt, 64)
```

Comment:
Why is uint64 parsing as a float here then converted back to uint64?

Reply:
This does look a little strange 😕

Reply:
Because there's no method to directly parse a `uint64` in the [strconv](https://golang.org/pkg/strconv/) package.

Reply:
I think this is where I was thrown off by the error log, It was failing to ParseFloat on a Int field...

Reply:
Why not use [strconv.ParseUint](https://golang.org/pkg/strconv/#ParseUint)?

Reply:
But the error itself was not related to that. It was because we were trying to parse an empty string.

Reply:
mmm... Actually... I think that [`ParseUint`](https://golang.org/pkg/strconv/#ParseUint) should do the job. I'm not sure why we are using this trick either.